backend/src/main/java/com/bakdata/conquery/io/mina/BinaryJacksonCoder.java

```diff
@@ -18,7 +18,7 @@ public class BinaryJacksonCoder implements CQCoder<NetworkMessage<?>> {
 	private final ObjectWriter writer;
 	private final ObjectReader reader;
 
-	public BinaryJacksonCoder(IdResolveContext datasets, Validator validator, ObjectMapper objectMapper) {
+	public BinaryJacksonCoder(NsIdResolver datasets, Validator validator, ObjectMapper objectMapper) {
```

Comment:
IdResolveContext konnte MetaId und NamespacedIds resolven. MetaIds werden aber gar nicht gebraucht in der Kommunikation zwischen Manager und Shards, deshalb gibt es hier nur einen NsIdResolver